lib/sentry/client_error.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Sentry.ClientError do
+  @moduledoc """
+  An exception struct that represents an error returned by Sentry.
+
+  This struct is designed to manage and handle errors originating from operations
+  in the client application. The `:reason` field signifies the cause of the error
```

Comment:
I think "client application" can be confusing?

```suggestion
  in the Sentry client. The `:reason` field signifies the cause of the error
```